Libraries/LibWeb/HTML/HTMLHyperlinkElementUtils.cpp

```diff
@@ -509,4 +512,54 @@ void HTMLHyperlinkElementUtils::follow_the_hyperlink(Optional<String> hyperlink_
     MUST(target_navigable->navigate({ .url = url_string, .source_document = hyperlink_element_utils_document(), .referrer_policy = referrer_policy, .user_involvement = user_involvement }));
 }
 
+void HTMLHyperlinkElementUtils::download_hyperlink(Optional<String> hyperlink_suffix, UserNavigationInvolvement user_involvement)
```

Comment:
Could you please add a spec link? As far as I understand, that's https://html.spec.whatwg.org/multipage/links.html#following-hyperlinks

Reply:
I added it.